.github/workflows/go.yml

```diff
@@ -0,0 +1,33 @@
+name: Go
```

Comment:
Please rename it to `[Check] go validation` and the file to `check-go-validation.yml` also, please remove all empty lines except line 8

Reply:
OK. this was what Github itself recommended me :D

Reply:
Yeah its just that we start to implement some conventions for the github workflow files and its better to have it like that from the beggining 😄